crates/router/src/connector/worldline/transformers.rs

```diff
@@ -1,13 +1,39 @@
+use std::collections::HashMap;
+
 use api_models::payments as api_models;
 use common_utils::pii::{self, Email};
+use error_stack::{IntoReport, ResultExt};
 use masking::{PeekInterface, Secret};
+use once_cell::sync::Lazy;
+use regex::Regex;
 use serde::{Deserialize, Serialize};
 
 use crate::{
     core::errors,
     types::{self, api, storage::enums},
 };
 
+static CARD_REJEX: Lazy<HashMap<CardProduct, Result<Regex, regex::Error>>> = Lazy::new(|| {
```

Comment:
1. Include links/references from where you included this regex.
2. Typo: `REGEX`.